### PR TITLE
[SLC2017] Announce SLC Dates for 2017

### PR DESCRIPTION
**Pull request by davidasnider**
We have dates and a venue.

### CODE REVIEW

**Comment by mattstratton** [comment]
You might want to update your `welcome.md` as it currently says you are still waiting for dates.

https://deploy-preview-1333--devopsdays-web.netlify.com/events/2017-salt-lake-city/welcome/

File: data/events/2017-salt-lake-city.yml [hunk 1]
```diff
@@ -5,17 +5,17 @@ status: "current" # Options are "past" or "current". Use "current" for upcoming.
...
 cfp_date_start:  # start accepting talk proposals.
 cfp_date_end:  # close your call for proposals.
 cfp_date_announce:  # inform proposers of status
 
 # Location
 #
-coordinates: "40.7631, -111.8828" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
-location: "Salt Lake City" # Defaults to city, but you can make it the venue name.
+coordinates: "40.552756, -111.901249" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Noah's Event Venue" # Defaults to city, but you can make it the venue name.
```
Comment (mattstratton) [inline data/events/2017-salt-lake-city.yml]: Is this the actual venue name?

**Comment by davidasnider** [comment]
Fixed all the welcome.md.    The name "Noah's Event Venue" is correct. http://www.noahseventvenue.com

**Comment by davidasnider** [comment]
I can either do a new pull request, or a separate one to fix this.   Also, the Venue name really is “Noah’s Event Venue”.  http://www.noahseventvenue.com <http://www.noahseventvenue.com/>

**Comment by bridgetkromhout** [comment]
> I can either do a new pull request, or a separate one to fix this.

Yeah, so since this was merged, if you still want changes, please submit a new one! Thanks. :)

**Comment by davidasnider** [comment]
Yep. This was an old response.  We're all good.

**Pull request merged**